### PR TITLE
[ALLUXIO-2952] Add unit test for RestUtils

### PR DESCRIPTION
**Pull request by uronce-cc**

### CODE REVIEW

**Comment by calvinjia** [comment]
Jenkins, test this please.

**Comment by uronce-cc** [comment]
Jenkins, test this please

File: core/server/common/src/test/java/alluxio/RestUtilsTest.java [hunk 1]
```diff
@@ -0,0 +1,94 @@
+package alluxio;
```
Comment (calvinjia) [inline core/server/common/src/test/java/alluxio/RestUtilsTest.java]: License is missing

File: core/server/common/pom.xml [hunk 2]
```diff
@@ -94,6 +94,24 @@
...
+      <groupId>org.glassfish.jersey.core</groupId>
+      <artifactId>jersey-server</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <!-- Required to decode json objects for rest API -->
+    <dependency>
+      <groupId>org.glassfish.jersey.media</groupId>
+      <artifactId>jersey-media-json-jackson</artifactId>
+      <scope>test</scope>
+    </dependency>
```
Comment (calvinjia) [inline core/server/common/pom.xml]: I would prefer not to bring in additional dependencies just for testing this, is there a simpler way?

Comment (uronce-cc) [inline core/server/common/pom.xml]: removed some dependencies, but jersey is needed

File: core/server/common/src/test/java/alluxio/RestUtilsTest.java (hunk 1 above)
Comment (uronce-cc) [inline core/server/common/src/test/java/alluxio/RestUtilsTest.java]: added

**Comment by uronce-cc** [comment]
jenkins, test this please

**Comment by uronce-cc** [comment]
@calvinjia any more comments?

**Comment by uronce-cc** [comment]
Jenkins, test this please

**Pull request merged**